addons/stock/wizard/stock_replenishment_info.xml

```diff
@@ -31,4 +37,43 @@
         <field name="context">{'default_orderpoint_id': active_id}</field>
         <field name="target">new</field>
     </record>
+
+    <record id="replenishment_option_tree_view" model="ir.ui.view">
+        <field name="name">stock.replenishment.option.tree.view</field>
+        <field name="model">stock.replenishment.option</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="qty_to_order" invisible="1"/>
+                <field name="warehouse_id" string="Warehouse"/>
+                <field name="location_id" string="Warehouse Location"/>
+                <field name="free_qty" decoration-danger="free_qty &lt; qty_to_order" string="Available Quantity"/>
+                <field name="uom" string="UoM"/>
+                <field name="route_id"/>
+                <field name="lead_time"/>
+                <button name="select_route" string="Select Route" type="object" class="btn btn-link"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="replenishment_option_warning_view" model="ir.ui.view">
+        <field name="name">stock.replenishment.warning.view</field>
+        <field name="model">stock.replenishment.option</field>
+        <field name="priority">100</field>
+        <field name="arch" type="xml">
+            <form>
+                <div>
+                    <field name="warehouse_id"/> can only provide <field name="free_qty"/> <field name="uom"/><br/>While the quantity to order is <field name="qty_to_order"/> <field name="uom"/>
```

Comment:
For other languages than English, the words order may differ to build this message. I'ts better to add a new char field on the `stock.replenishment.option` with `{1}, {2}, ...` placeholders, so the translated message is easily built too

Reply:
@R1D1CUL0US could you also do this comment finally? William convince me that it's necessary.